pkg/object/mqttproxy/broker.go

```diff
@@ -134,16 +136,21 @@ func newBroker(spec *Spec, store storage, muxMapper context.MuxMapper, memberURL
 	if spec.TopicCacheSize <= 0 {
 		spec.TopicCacheSize = 100000
 	}
-	broker.topicMgr = newTopicManager(spec.TopicCacheSize)
+	broker.topicMgr = newTopicManager(spec)
 	broker.sessMgr = newSessionManager(broker, store)
 	broker.connectionLimiter = newLimiter(spec.ConnectionLimit)
 	go broker.run()
-	ch, closeFunc, err := broker.sessMgr.store.watchDelete(sessionStoreKey(""))
-	if err != nil {
-		logger.SpanErrorf(nil, "get watcher for session failed, %v", err)
-	}
-	if ch != nil {
-		go broker.watchDelete(ch, closeFunc)
+	if !spec.BrokerMode {
+		ch, closeFunc, err := broker.sessMgr.store.watchDelete(sessionStoreKey(""))
```

Comment:
I don't understand why the feature of kicking off sessions in broker mode is disabled.